package.json

```
@@ -34,7 +34,7 @@
     "watch-live-reload": "./node_modules/.bin/grunt watch",
     "lint": "./node_modules/.bin/standard --fix",
     "lint-sass": "stylelint '**/*.scss'",
-    "test": "rm -rf ./pacts && NODE_ENV=test ./node_modules/mocha/bin/mocha --exclude **/*.cy.test.js '!(node_modules)/**/*.test'.js",
+    "test": "rm -rf ./pacts && NODE_ENV=test ./node_modules/mocha/bin/mocha --exclude **/*.cy.js '!(node_modules)/**/*.test'.js",
     "test:no-pact": "rm -rf ./pacts && NODE_ENV=test ./node_modules/mocha/bin/mocha --exclude '**/*+(.cy|.pact)'.test.js '!(node_modules)/**/*.test'.js",
```

This script can be updated too, as we no longer need to exclude files with "cy.test.js" extension

```suggestion
    "test:no-pact": "rm -rf ./pacts && NODE_ENV=test ./node_modules/mocha/bin/mocha --exclude **/*.pact.test.js '!(node_modules)/**/*.test'.js",
```